common/pb/event_notify_user.proto

```diff
@@ -0,0 +1,18 @@
+import "session_event.proto";
+
+message Event_NotifyUser {
+
+	enum NotificationType {
+        PROMOTIONSUCCESS = 1;
+        PROMOTED = 2;
```

Comment:
We might be able to not use this entire container if the server can just send the already connected user a new ServerInfo_User for themselves.